.github/workflows/coverage.yaml

```diff
@@ -39,12 +38,19 @@ jobs:
       - name: Generate code coverage
         run: |
           ./scripts/tarpaulin.sh
+      
+      - name: List files after tarpaulin run
+        run: |
+          echo "Checking files in protocols directory"
```

Comment:
What is this checking? 
could we also do `ls -l protocols && ls -l roles && ls -l utils` ?

Reply:
This is an outdated snapshot

Reply:
It was used to check if the xml files were generating or not...

Reply:
Umm you got my confused (: 

Introducing changes in a commit and then reverting them in the next is not recommended. can you please fix up the second commit?